src/app/routines/routineCard/RoutineCard.scss

```diff
@@ -85,4 +84,12 @@
       cursor: pointer;
     }
   }
+}
+
+@media (max-width: 1690px) {
+  .routine-list-item {
+    width: auto;
```

Comment:
it makes no sence to introduce that. It will probably have the same output as previous "bug"
Cause it's auto width, so components will have same problems that their width will be based on description size:
from 180-500px

Reply:
nope, they are being scaled to fulfil the whole width of the container